bapt/cli.py

```diff
@@ -24,16 +24,29 @@ def main():
     parser.add_argument('-f', '--filename', default=None,
                         help='Path to file containing alignment information.')
     parser.add_argument('-n', '--name',
-                        help='List of compound names (comma seperated).' +
-                        'Must be used in conguction with --ip and --ea.')
+                        help='List of compound names (comma seperated). ' +
+                        'Must be used in conguction with --ip and --ea. ' +
+                        'Use $_x$ and $^y$ for subscript and superscript.')
     parser.add_argument('-i', '--ip',
                         help='List of ionisation potentials (comma separated).')
     parser.add_argument('-e', '--ea',
                         help='List of electron affinities (comma separated).')
+    parser.add_argument('-cbo', '--cbo', default=None,
```

Comment:
Abbreviated arguments should only be one letter long. I.e., if the long argument is `--cbo`, the short one should be something like `-c`. If `-c` is already taken by another option, you can either come up with a different letter or leave it as a long argument only.

So this option, vbo and eg, should should be:

```
-c, --cbo
-v, --vbo
-e, --eg
```